Documentation/introduction/resources.rst

```diff
@@ -20,3 +20,7 @@ Here's a list of Apache NuttX resources that you might find helpful:
    * `Git Cheat Sheet (by GitHub) <https://github.github.com/training-kit/downloads/github-git-cheat-sheet.pdf>`_
    * `Git Book (online) <https://git-scm.com/book/en/v2>`_
 
+Legacy README
```

Comment:
Unfortunately this won't work. latexpdf fails when including large files. We can simply provide a link to the old README without including it.

Reply:
@raiden00pl I think the issue is not because the file is too big, but because there is a Verbatim line too big:
"Dimension too large"

Reply:
That was Sphinx interpreting the `.md `file as a `.rst` file. It now includes properly.
It looks like this on PDF:
![image](https://github.com/apache/nuttx/assets/37779557/62548532-9620-48f1-ae58-4cbee2241b57)